INSTALL.md

```diff
@@ -1,4 +1,4 @@
-Building Mergin Maps mobile app from source - step by step
+Building Mergin Maps Mobile from source - step by step
```

Comment:
Officially, we call it `Mergin Maps mobile app`, you can find the names here: https://docs.google.com/document/d/1Oot50Wk11HW2sAIiQqSNGiQhNmkC8k-32NV8vdxoFws/edit?usp=sharing. When you do not need to refer to a specific component, we just say "Mergin Maps" (not this case).

Reply:
In short, we just say "the app". E.g. "Steps to build and run the app:"